extensions/wikia/ArticleVideoMVP/styles/premium-mvp.scss

```diff
@@ -0,0 +1,135 @@
+#premium-mvp-article-header {
+	display: block;
+}
+
+#premium-mvp-video {
+	background-position: center center;
+	background-size: cover;
+	cursor: pointer;
+	margin-bottom: 11px;
+	padding-top: 56%;
```

Comment:
That's the only way to have height dependent on width and keep the ratio. Actually I have to make this value more precise as it defines 16:9 ratio - it should be `56,25%`.